tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/Steps.cs

```diff
@@ -560,7 +561,7 @@ private async Task SetIdAsync(string query)
             var id = await StepsHelper.GetAsync<Guid>(
                 _connectionStringProvider.Value.GetConnectionString(), query, _cancellationToken.Value);
 
-            id.ShouldNotBeNull();
+            id.ToString().IsNullOrEmpty().ShouldBeFalse();
```

Comment:
What was the reason for this change? It looks like you ended up using an extension method from `Castle.Core.Internal` here -- which is undesirable.

Reply:
fixed it . Thanks